packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -440,9 +461,29 @@ export function parseExecutionPayload(data: ExecutionPayloadRpc): allForks.Execu
     baseFeePerGas: quantityToBigint(data.baseFeePerGas),
     blockHash: dataToBytes(data.blockHash, 32),
     transactions: data.transactions.map((tran) => dataToBytes(tran)),
+    ...withdrawalsAttr,
+  };
+}
+
+function serializeWithdrawal(withdrawal: capella.Withdrawal): WithdrawalV1 {
+  return {
+    index: numToQuantity(withdrawal.index),
+    validatorIndex: numToQuantity(withdrawal.validatorIndex),
+    address: bytesToData(withdrawal.address),
+    recipient: bytesToData(withdrawal.address),
+    amount: numToQuantity(withdrawal.amount),
   };
 }
 
+function deserializeWithdrawal(serialized: WithdrawalV1): capella.Withdrawal {
+  return {
+    index: quantityToNum(serialized.index),
+    validatorIndex: quantityToNum(serialized.validatorIndex),
+    address: dataToBytes(serialized.recipient ?? serialized.address),
+    amount: quantityToBigint(serialized.amount),
```

Comment:
note to self : convert to gwei + check endianess
https://github.com/ethereum/execution-apis/pull/195/files#diff-571851156237b81c20006ce9d7a70b3421f19540cead91f45955316c52fe94c0R202